src/store/reducers/deploymentsReducer.js

```diff
@@ -89,24 +121,57 @@ const deleteDeploymentReducer = (state, { payload, fromAction }) => {
 const updateDeploymentsReducer = (state, { payload, fromAction }) => {
   const { entities: { deployments }, result } = normalize(payload, deploymentListSchema);
   return update(state, {
-    entities: { $set: deployments },
+    entities: { deployments: { $set: deployments } },
     items: { $set: result },
     lastUpdated: { $set: moment() },
     ...setPending(fromAction.type, false)
   });
 };
 
+const updateDeploymentReducer = (state, { payload, fromAction }) => {
+  const deviceStatuses = dot.pick('deviceStatuses', payload);
```

Comment:
Is dot.pick needed for a single level?

Reply:
It looks better than a ternary operator.